tests/integration/cli/batch.py

```diff
@@ -24,79 +24,39 @@ def test_batch_run(self):
         '''
         Tests executing a simple batch command to help catch regressions
         '''
-        ret = ['',
-               "Executing run on ['sub_minion']",
-               '',
-               'sub_minion:',
-               'retcode:',
-               '    0',
-               '    batch testing',
-               '',
-               "Executing run on ['minion']",
-               '',
-               'minion:',
-               'retcode:',
-               '    0',
-               '    batch testing']
+        ret = 'Executing run on [\'sub_minion\']'
         cmd = self.run_salt('\'*\' test.echo \'batch testing\' -b 50%')
         if six.PY3:
-            self.assertCountEqual(cmd, ret)
+            self.assertIn(ret, cmd)
```

Comment:
If we're just doing the same `assertIn` here, we don't need to split this up between `six.PY3` anymore.